mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/presenter/LinkBankAccountViewModel.kt

```diff
@@ -0,0 +1,118 @@
+package org.mifos.mobilewallet.mifospay.bank.presenter
+
+import android.content.Context
+import android.content.Intent
+import android.os.Handler
+import androidx.lifecycle.ViewModel
+import com.mifos.mobilewallet.model.domain.BankAccountDetails
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import org.json.JSONObject
+import org.mifos.mobilewallet.mifospay.R
+import org.mifos.mobilewallet.mifospay.common.Constants
+import org.mifos.mobilewallet.mifospay.domain.model.Bank
+import org.mifos.mobilewallet.mifospay.utils.FileUtils
+import java.util.Random
+import javax.inject.Inject
+
+
+@HiltViewModel
+class LinkBankAccountViewModel @Inject constructor() : ViewModel() {
```

Comment:
Inject `LocalAssetRepository` and fetch banks list from there 
<img width="1728" alt="image" src="https://github.com/openMF/mobile-wallet/assets/5724482/c9d6ffb5-0142-452a-af7c-87f67f328d33">

Reply:
Here i was getting an error changed the json from {"banks": []} to only []
![Screenshot (318)](https://github.com/openMF/mobile-wallet/assets/91717339/a157cbd9-1dc9-44b1-97b0-fda93a365c33)